easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.13-foss-2021a-R-4.1.0.eb

```diff
@@ -1047,9 +1050,6 @@ exts_list = [
     ('GenABEL', '1.8-0', {
         'checksums': ['434fc806b918738ec67171a37a47099d1706332819317e368fe0ccb8f149a469'],
     }),
-    ('JASPAR2020', '0.99.10', {
```

Comment:
@SIB-software It seems like this was removed unintentionally?

Reply:
Ahh! It was not based on the last file version.
Should be fixed now